lib/ansible/modules/remote_management/ucs/ucs_ntp_server.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_ntp_server
+short_description: Configures NTP server on Cisco UCS Manager
+extends_documentation_fragment:
+- ucs
+description:
+- Configures NTP server on Cisco UCS Manager.
+- Examples can be used with the L(UCS Platform Emulator,https://communities.cisco.com/ucspe).
+options:
+  state:
+    description:
+    - If C(absent), will remove an ntp server.
+    - If C(present), will add or update an ntp server.
+    choices: [absent, present]
+    default: present
+
+  name:
+    description:
+    - The ntp server ip address or hostname.
+    - Enter up to 63 characters that form a valid hostname.
+    - Enter a valid IPV4 Address.
+
+  description:
+    description:
+    - A user-defined description of the ntp server.
+    - Enter up to 256 characters.
+    - "You can use any characters or spaces except the following:"
+    - "` (accent mark), \ (backslash), ^ (carat), \" (double quote), = (equal sign), > (greater than), < (less than), or ' (single quote)."
+    aliases: [ descr ]
+    default: ""
+
+requirements:
+- ucsmsdk
+author:
+- John McDonough (@movinalot)
+- CiscoUcs (@CiscoUcs)
+version_added: "2.7"
+'''
+
+EXAMPLES = r'''
+- name: Configure ntp server
+  ucs_ntp_server:
+    hostname: 172.16.143.150
+    username: admin
+    password: password
+    name: 10.10.10.10
+    description: Internal NTP Server by IP address
+    state: present
+
+- name: Configure ntp server
+  ucs_ntp_server:
+    hostname: 172.16.143.150
+    username: admin
+    password: password
+    name: pool.ntp.org
+    description: External NTP Server by hostname
+    state: present
+
+- name: Remove ntp server
+  ucs_ntp_server:
+    hostname: 172.16.143.150
+    username: admin
+    password: password
+    state: absent
```

Comment:
I would move the state one line below. First the selector, than the action, is what I usually do.

Reply:
Changed